packages/errors/index.js

```diff
@@ -49,9 +53,10 @@ if (missing.length > 0) {
 const {
   bare,
   details: redacted,
```

Comment:
```suggestion
  details: X,
```

Reply:
Or better yet, normalize `assert.X` as the replacement for a deprecated `assert.details`, like this PR does for `assert.makeError` vs `assert.error`.

Need not be in this PR but I'd at least like your opinion ;)